forge/db/controllers/ProjectTemplate.js

```diff
@@ -196,7 +196,10 @@ module.exports = {
             }
         }
         if (typeof result.httpNodeAuth?.pass === 'string' && result.httpNodeAuth.pass.length > 0) {
-            result.httpNodeAuth.pass = hash(result.httpNodeAuth.pass)
+            if (result.httpNodeAuth.pass.length !== 42 && !result.httpNodeAuth.pass.startsWith('$2b$10$')) {
+                // only hash if not already hashed
```

Comment:
Reading just to code the context might be nice to have.

```suggestion
                // only hash if not already hashed, see also: https://github.com/FlowFuse/flowfuse/issues/4960
```